ldes-server-port-publication-rest/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/rest/converters/JsonLDConverter.java

```diff
@@ -0,0 +1,63 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.rest.converters;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.entities.LdesFragment;
+import be.vlaanderen.informatievlaanderen.ldes.server.rest.services.JSONObjectCreatorIml;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.springframework.http.HttpInputMessage;
+import org.springframework.http.HttpOutputMessage;
+import org.springframework.http.MediaType;
+import org.springframework.http.converter.HttpMessageConverter;
+import org.springframework.http.converter.HttpMessageNotReadableException;
+import org.springframework.http.converter.HttpMessageNotWritableException;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.List;
+
+public class JsonLDConverter implements HttpMessageConverter<LdesFragment> {
```

Comment:
This is an example of why I prefer the old java-style convention for class naming over the Spring one. This class should be named JsonLdConverter according to the convention we're working with, as opposed to the old style JSONLDConverter, but not a mix of the two. Please rename.

Reply:
Updated name to JsonLdConverter